st2api/st2api/cmd/api.py

```diff
@@ -25,6 +25,7 @@
 from st2common.service_setup import teardown as common_teardown
 from st2common.util.monkey_patch import monkey_patch
 from st2api import config
+config.register_opts()
```

Comment:
It was already there before, I just accidentally removed it during stream changes.

I do agree that it's awful though.